frontend/src/components/ui/Switch/switch.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import * as SwitchPrimitives from '@radix-ui/react-switch';
+
+import { cn } from '@/lib/utils';
```

Comment:
can we add proptypes for this refer button component

Reply:
done